tests/DevApps/ciam/CiamTest.sln

```diff
@@ -0,0 +1,28 @@
+﻿
```

Comment:
remove space

Reply:
false postive. This is the format for sln files.